### PR TITLE
Feature/7925 security alternate sign in accounts 

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

File: src/app/account-settings/components/dialog-security-alternate-account-delete/dialog-security-alternate-account-delete.component.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      i18n="@@account.unlink"
+    >
+      Unlink account
+    </button>
+    <button
+      mat-stroked-button
+      color="primary"
+      (click)="cancel()"
+      i18n="@@shared.cancel"
+      id="cancel-emails-button"
```
Comment (DanielPalafox) [inline src/app/account-settings/components/dialog-security-alternate-account-delete/dialog-security-alternate-account-delete.component.html:27]: Could you remove the id thats not been used

File: src/app/account-settings/components/settings-security-alternate-sign-in/settings-security-alternate-sign-in.component.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          <mat-icon>delete</mat-icon>
+        </button>
+      </td>
+    </ng-container>
+
+    <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+    <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
+  </table>
+  <div *ngIf="!accounts.length" class="emptyAccounts">
+    <mat-divider class="emptyAccounts"></mat-divider>
```
Comment (DanielPalafox) [inline src/app/account-settings/components/settings-security-alternate-sign-in/settings-security-alternate-sign-in.component.html:58]: is this mat-divider correct having the same class as the parent div

File: src/app/account-settings/components/settings-security-alternate-sign-in/settings-security-alternate-sign-in.component.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
+<app-settings-panels-data *ngIf="accounts$ | async as accounts">
+  <div i18n="@@account.youCanSingIntoOrcid">
+    You can sign into ORCID using the personal and institutional accounts you
+    have linked to your ORCID record.
+  </div>
+
+  <a
+    rel="noopener noreferrer"
```
Comment (DanielPalafox) [inline src/app/account-settings/components/settings-security-alternate-sign-in/settings-security-alternate-sign-in.component.html:8]: Could you add the missing blank target

File: src/locale/properties/account/account.rl.properties [hunk 4]
```diff
@@ -1,4 +1,4 @@
-account.emailFrequency=RL
+saccount.emailFrequency=RL
```
Comment (DanielPalafox) [inline src/locale/properties/account/account.rl.properties:1]: Could you fix this typo

File: src/app/types/account-alternate-sign-in.endpoint.ts [hunk 5]
```diff
@@ -0,0 +1,37 @@
+export interface SocialAccount {
+  dateCreated: number
+  lastModified: number
+  id: SocialAccountId
+  accesstoken: string
+  displayname: string
+  email: string
+  expiretime: number
+  imageurl?: null
```
Comment (DanielPalafox) [inline src/app/types/account-alternate-sign-in.endpoint.ts:9]: Could you please define the missing interface types

File: src/app/account-settings/components/settings-security-alternate-sign-in/settings-security-alternate-sign-in.component.html (hunk 2 above)
Comment (leomendoza123) [reply src/app/account-settings/components/settings-security-alternate-sign-in/settings-security-alternate-sign-in.component.html:58]: yes that is what we need there... but I am changing the name so is not confusing

**Review by DanielPalafox — APPROVED** [review]

**Comment by leomendoza123** [comment]
[Security: Alternate sign-in accounts (3)](https://trello.com/c/vgjrkVud/7925-security-alternate-sign-in-accounts-3)

**Pull request merged**